addons/mail/static/tests/composer/composer.test.js

```diff
@@ -1160,3 +1160,20 @@ test("composer reply-to message is restored page reload", async () => {
     await openDiscuss(channelId);
     await contains(".o-mail-Composer:contains('Replying to')");
 });
+
+test.tags("html composer");
+test("html composer: basic rendering", async () => {
+    await startServer();
+    await start();
+    const composerService = getService("mail.composer");
+    await openFormView("res.partner", serverState.partnerId);
+    await click("button", { text: "Send message" });
+    composerService.setTextComposer();
```

Comment:
Isn't this the default state? It's strange to start by doing this interaction. I'd expect first to check the default (input) then switch to html, and if you want to test this function too, then check to re-set to text at the end.

Reply:
This needs to be done in a way for each test to be isolated. Isn't local storage somehow already mocked in tests to prevent this kind of issues?

Reply:
Also the current test doesn't reset to setTextComposer at the end, so if the issue really exists, following tests would be impacted too?

Reply:
so in this case, we dont have to switch it back to text composer in the end of the test.

Reply:
Well it's important to test we can go back to a normal and functioning input too.